_episodes/02-git.md

```diff
@@ -533,7 +535,8 @@ where `branch_name` is the name of the branch you would like to merge.
 We can merge our `doc_update` branch to get changes on our `main` branch:
 
 ~~~
-$ get merge doc_update
+$ git checkout main
```

Comment:
Should this be at line 522 instead? No strong preference.

Reply:
I'm sorry but I don't follow, line 522-523 is this:

> In this case, we will first merge the changes from our `doc_update` branch,
> then our `zen` branch, so we should be on our `main` branch.

Did you mean `git checkout main` should be moved to that line?

Reply:
I meant that this `git checkout` illustrates the instruction "When you do a merge, you should be on the branch you would like to merge into." on line 521.

Reply:
Ah I see, it took me a while to understand. The thing is line 520-531 is for explanatory purpose:
```
To incorporate these changes in `main`, you will need to do a `git merge`.
When you do a merge, you should be on the branch you would like to merge into.
In this case, we will first merge the changes from our `doc_update` branch,
then our `zen` branch, so we should be on our `main` branch.
Next we will use the `git merge` command.

The syntax for this command is 

~~~
$ git merge branch_name
~~~
{: .language-bash}

where `branch_name` is the name of the branch you would like to merge.
```
Then after that comes the instruction part:
```
We can merge our `doc_update` branch to get changes on our `main` branch:

~~~
$ git checkout main
$ git merge doc_update
~~~
{: .language-bash}
```
Of course the `git checkout main` command could also be added to line 522 (though I prefer not to for the sake of conciseness of the explanation), but it should still be in the instruction part too.

Reply:
I think it fits where it is currently placed in this PR.